### ISSUE TITLE
Installation fails with ember 1.13

### ISSUE THREAD

**Issue by stevebest** [mike-north/ember-material-lite#33]
```
$ ember new mdl && cd mdl
...
...coffee break!

$ ember install ember-material-lite
version: 1.13.1
Installed packages for tooling via npm.
installing ember-material-lite
The `ember generate` command requires an entity name to be specified. For more details, use `ember help`.
```

**Comment by sunil-shrestha** [comment]
Here is the similar issue. It should help.
https://github.com/mike-north/ember-material-lite/issues/29

**Comment by stevebest** [comment]
Hm, it did help. Thanks, @sunil-shrestha!

Still, for a new user this issue causes a lot of confusion, so it should either be fixed somehow or documented in installation guide. Leaving it open.

**Comment by mike-north** [comment]
should be fixed by #52
